doc/next/_sources/reference/post-metrics-data.rst.txt

```diff
@@ -15,19 +24,56 @@ POST /metrics/data
  * - Content-Type
    - ``application/sellside.metrics.data-v1+json``
 
-This URL returns the requested reporting metrics in a format described in the new :ref:`metrics_reporting` API.
+This URL returns the requested reporting metrics in a format described in the new :ref:`metrics_reporting_v1` API.
+
+Query and Response
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
have a look at some other sellside endpoint that has multiple versions; the order is typically the most recent version on top